multigeiger/display.cpp

```diff
@@ -106,28 +106,33 @@ void DisplayGMC(int TimeSec, int RadNSvph, int CPS, bool use_display, bool conne
   if (!isLoraBoard) {
     char output[80];
     int TimeMin = TimeSec / 60;         // calculate number of minutes
-    if (TimeMin >= 999) TimeMin = 999;  // limit minutes to max. 999
+    TimeMin = TimeMin % 999;  // limit minutes to max. 999, roll over
 
     // print the upper line including time and measured radation
+    pu8x8->setFont(u8x8_font_open_iconic_embedded_1x1);
+    sprintf(output, "%c", 0x4A);
+    if (ble_active) pu8x8->print(output);
```

Comment:
how about removing the sprintf and the if and using:

```
pu8x8->print(ble_active ? '\x4A' : '\x20')
```

That would output 1 character in both cases so that the rest of the layout stays the same and nothing shifts around.

Reply:
That was actually my first version, then I tended towards giving more space for high counts if needed. If the shifting is not wanted, I'll revert to that first version.